lua/bufferline.lua

```diff
@@ -51,13 +52,17 @@ function bufferline.setup(user_config)
     function(tbl)
       local index = tonumber(tbl.args)
       if not index then
-        return notify('Invalid argument to `:BufferGoto`', vim.log.levels.ERROR)
+        return notify(
+          'Invalid argument to ' .. utils.markdown_inline_code':BufferGoto',
+          vim.log.levels.ERROR
+        )
       end
+
       api.goto_buffer(index)
     end,
     {
       complete = function()
-        local buffers = require'bufferline.state'.buffers
```

Comment:
The local `require` went against our conventions, so I put a `local state = …` at the top instead